pages/sign-up.html

```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
```

Comment:
p1. title이 회원가입으로 되어있지 않네요. 수정 부탁드립니다!

```suggestion
 <title>회원가입</title>
```